lib/kollus.rb

```diff
@@ -51,13 +60,14 @@ def token; @token end
   end
 
 
-  def upload(title = nil, expire_time = 600, encrypted = true, audio = false)
+  def upload(category_key = nil, title = nil, expire_time = 600, encrypted = true, audio = false)
+    tries ||= 3
     api_uri = URI('http://api.kr.kollus.com/0/media_auth/upload/create_url.json?access_token=' + @token)
     params = {
       # 값의 범위는 0 < expire_time <= 21600 입니다. 빈값을 보내거나 항목 자체를 제거하면 기본 600초로 설정됩니다.
       expire_time: expire_time,
       # 업로드한 파일이 속할 카테고리의 키(API를 이용하여 확득 가능)입니다. 빈값을 보내 거나 항목 자체를 제거하면 '없음'에 속합니다.
-      category_key: 'bw2z2ngbkatkqx77',
```

Comment:
It's okay. I think it is useless unless one knows the api key corresponding to the category.